### PR TITLE
Backrest: Add version 1.5.0

### LINKED ISSUE #13978: [Request]: Backrest
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

Backrest

### Description

Web UI and orchestrator for restic backup.

### Homepage

https://github.com/garethgeorge/backrest

### Download Link(s)

https://github.com/garethgeorge/backrest/releases

### Some Indication of Popularity/Repute

34 fork 🍴 and 1.1k stars ⭐

### PR DESCRIPTION
**Pull request by Chucky2401**
Adds a manifest for the Backrest software. Tested with a local and global installation and uninstallation. Also checked the manifest updates automatically.

Closes #13978 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/backrest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            "64bit": {
+                "url": "https://github.com/garethgeorge/backrest/releases/download/v$version/backrest_Windows_x86_64.zip"
+            },
+            "arm64": {
+                "url": "https://github.com/garethgeorge/backrest/releases/download/v$version/backrest_Windows_arm64.zip"
+            }
+        },
+        "hash": {
+            "url": "",
+            "regex": "$sha256\\s+$basename"
```
Comment (aliesbelik) [inline bucket/backrest.json:38]:
```suggestion
            "url": "$baseurl/backrest_$version_checksums.txt"
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]

File: bucket/backrest.json (hunk 1 above)
Comment (Chucky2401) [reply bucket/backrest.json:38]: Should I make the necessary changes?
This is my first PR and I'm a bit confused.

**Comment by Chucky2401** [comment]
I made the modification on my side, and I didn't know any change will be pushed to the PR 😅.

**Review by aliesbelik — APPROVED** [review]

**Comment by aliesbelik** [comment]
Please update to the latest version, otherwise looks good.

**Comment by Chucky2401** [comment]
Updated to the latest version

**Pull request merged**